src/SprykerEco/Zed/Heidelpay/HeidelpayDependencyProvider.php

```diff
@@ -25,30 +25,103 @@ class HeidelpayDependencyProvider extends AbstractBundleDependencyProvider
 
     public const SERVICE_UTIL_ENCODING = 'util encoding service';
 
+    public const PLUGIN_HEIDELPAY_NOTIFICATION_EXPANDER = 'PLUGIN_HEIDELPAY_NOTIFICATION_EXPANDER';
+
     /**
      * @param \Spryker\Zed\Kernel\Container $container
      *
      * @return \Spryker\Zed\Kernel\Container
      */
     public function provideBusinessLayerDependencies(Container $container): Container
+    {
+        $container = parent::provideBusinessLayerDependencies($container);
+        $container = $this->addCurrencyFacade($container);
+        $container = $this->addMoneyFacade($container);
+        $container = $this->addSalesFacade($container);
+        $container = $this->addSalesQueryContainer($container);
+        $container = $this->addUtilEncodingService($container);
+        $container = $this->addHeidelpayNotificationExpanderPlugins($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideCommunicationLayerDependencies(Container $container): Container
+    {
+        $container = parent::provideCommunicationLayerDependencies($container);
+        $container = $this->addSalesFacade($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addSalesFacade(Container $container): Container
+    {
+        $container[static::FACADE_SALES] = function (Container $container) {
```

Comment:
Could you use new container syntax? Also don't forget to raise `kernel` version dependency for it